pkg/tasks/telemeter.go

```diff
@@ -35,134 +35,9 @@ func NewTelemeterClientTask(client *client.Client, factory *manifests.Factory, c
 }
 
 func (t *TelemeterClientTask) Run() error {
-	if t.config.IsEnabled() {
-		return t.create()
-	}
-
 	return t.destroy()
```

Comment:
just a very small nit (we can do it also as a follow-up). let's document why we are "just" destroying telemeter client here, namely to catch the upgrade path to remote-write. it could be non-obvious when someone looks in here only.